.github/workflows/build-assets.yml

```diff
@@ -372,6 +372,11 @@ jobs:
     strategy:
       matrix:
         include:
+          - os: "windows-2022"
+            mono: ""
+            sudo: ""
+            cmake_config_options: '-T host=x64 -A Win32 -DUSE_OPTIONAL=OFF -DCLIENTTOOLS_ONLY=ON -DUSE_AZURE=OFF -DUSE_CASSANDRA=OFF -DUSE_JAVA=OFF -DUSE_OPENLDAP=OFF'
```

Comment:
Generally I don't specify the `-T host=x64` and let it use the default (which may well be the x64 toolchain)`.  I suspect this is 100% fine, just never tried it...